### PR TITLE
[WIP] PIM-5000: do not apply changes on unchecked products

### PR DESCRIPTION
**Pull request by MarieMinasyan**
| Q                 | A
| ----------------- | ---
| Specs             | no regression
| Behats            | yes
| Blue CI           | running
| Changelog updated | yes
| Review and 2 GTM  | yes

### CODE REVIEW

File: src/Pim/Bundle/DataGridBundle/Extension/MassAction/MassActionDispatcher.php [hunk 1]
```diff
@@ -99,10 +99,12 @@ public function getRawFilters(Request $request)
         }
 
         if (true === $parameters['inset']) {
-            $productIds = $parameters['values'];
-            $filters = [['field' => 'id', 'operator' => 'IN', 'value' => $productIds]];
+            $filters = [['field' => 'id', 'operator' => 'IN', 'value' => $parameters['values']]];
```
Comment (jmleroux) [inline src/Pim/Bundle/DataGridBundle/Extension/MassAction/MassActionDispatcher.php]: no need to check if `$parameters['inset']` and `$parameters['values']` are set ?

**Comment by jmleroux** [comment]
Missing description array in the description : https://github.com/akeneo/pim-community-dev/blob/master/CONTRIBUTING.md

;)

**Comment by jmleroux** [comment]
missing behat test

**Comment by solivier** [comment]
missing behats and specs

File: src/Pim/Bundle/DataGridBundle/Extension/MassAction/MassActionDispatcher.php (hunk 1 above)
Comment (MarieMinasyan) [inline src/Pim/Bundle/DataGridBundle/Extension/MassAction/MassActionDispatcher.php]: no need as the method that defines this values does : 
return [
            'datagrid'   => $datagrid,
            'massAction' => $massAction,
            'inset'      => $inset,
            'values'     => $values
        ];

**Comment by jjanvier** [comment]
GTM

**Comment by jmleroux** [comment]
GTM

**Pull request merged**